src/main/java/com/lothrazar/cyclic/enchant/EnchantMagnet.java

```diff
@@ -69,7 +71,8 @@ public void onEntityUpdate(LivingUpdateEvent event) {
     }
     //Ticking
     int level = getLevelAll(entity);
-    if (level > 0) {
+    boolean solegnoliaPresent = ModList.get().isLoaded("botania") && BotaniaAPI.instance().hasSolegnoliaAround(entity);
```

Comment:
Also much easier than I expected it to be, nice!

Reply:
That does mean that Botania will always be present in dev, not sure if that matters to you.